app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,75 @@
+class TweetController < ApplicationController
+	get "/tweets" do 
+	  	if !!session[:id]
+	  		erb :"tweets/tweets"
+	  	else
+	  		redirect "/login"
+	  	end
+  	end
+
+  	get "/tweets/new" do 
+  		if !!session[:id]
+  			erb :"tweets/create_tweet"
+  		else
+  			redirect "/login"
+  		end
+  	end
+
+  	post "/tweets" do 
+  		user = User.all.find(session["id"])
+		if params[:content] == ""
+			redirect "/tweets/new"
+		else
+			user.tweets << Tweet.create(params)
+			redirect "/tweets"
+		end
+  	end
+
+  	get "/tweets/:id" do 
+  		if !!session[:id]
+	  		@tweet = Tweet.find(params[:id])
+	  		erb :"tweets/show_tweet"
+	  	else
+	  		redirect "/login"
+	  	end
+  	end
+
+  	get "/tweets/:id/edit" do
+  		if !!session[:id]
+  			@tweet = Tweet.find(params[:id])
```

Comment:
Shouldn't you verify a tweet belongs to the `current_user` (and not just that any user is logged in) before allowing them to edit?